h2o-py/tests/testdir_algos/glm/pyunit_GH_6722_glm_coefNames.py

```diff
@@ -0,0 +1,33 @@
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator as glm
+
```

Comment:
```suggestion


```
Ideally, I would try to stick to PEP-8 and [keep 2 blank lines between top-level functions](https://peps.python.org/pep-0008/#blank-lines).